gateway-service-definitions/src/main/resources/services/cm-ui/1.0.0/rewrite.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<rules>
+  <rule dir="IN" name="CM-UI/cm-ui/ui/inbound" pattern="*://*:*/**/cmf/{xxx_unique_path=**}?{**}">
+    <rewrite template="{$serviceUrl[CM-UI]}/cmf/{xxx_unique_path=**}?{**}"/>
+  </rule>
+  <rule dir="IN" name="CM-UI/cm-ui/ui/root" pattern="*://*:*/**/cmf/">
+    <rewrite template="{$serviceUrl[CM-UI]}/cmf/"/>
+  </rule>
+  <rule dir="IN" name="CM-UI/cm-ui/static/inbound" pattern="*://*:*/**/static/{xxx_unique_path=**}?{**}">
```

Comment:
why is the context path /static here shouldn't it be /cmf/static ?

Reply:
As discussed this is OK.